README.md

```diff
@@ -213,6 +213,26 @@ Keep the console open while testing. If you close it, the tunnel also closes and
       - For the **ClientSecret** key, replace *ENTER_YOUR_SECRET* with the key of your registered Azure application. Note that in production apps you should always use certificates as your application secrets, but for this sample we will use a simple shared secret password.
       - For the **Tenantd** key, replace *ENTER_YOUR_TENANT_ID* with your tenant ID, this information can be found on the application registration page.  
 
+1. Set the environment variable for DefaultAzureCredential.
+
+   Follow one of the examples below depending on your operating system to create the environment variable. If using Windows close your open IDE or shell and restart it to be able to read the environment variable.

```

Comment:
could we use dotnet secrets instead and provide a single, cross-platform, set of commands?
https://www.twilio.com/blog/2018/03/protecting-your-passwords-with-the-net-secret-manager.html

Reply:
if we use dotnet secrets instead, we could specified the credential. These credentials dont need commands to set environment variables.